src/styles/blocks/product-card.scss

```diff
@@ -0,0 +1,95 @@
+.product-card {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: 198px;
+  background: $card-background-color;
+  border: 1px solid $card-border;
+  border-radius: 5px;
```

Comment:
You should set correct paddings for the card, and don't set width property to all card elements